test/expression_test.ts

```diff
@@ -92,4 +92,28 @@ describe('expressions', () => {
     expectTranslate('foo.bar;').to.equal('foo.bar;');
     expectTranslate('foo[bar];').to.equal('foo[bar];');
   });
+  it('should cast numeric values to int when bitwise operators are used for a binary expression',
```

Comment:
please remove this test, it's entirely redundant with the binary operator table test above.